plugins/rich-text/README.md

```diff
@@ -0,0 +1,98 @@
+# Builder.io example plugin
```

Comment:
would be good to update the reamde to explain what this is. Does need a lot of updates, just mention this is an example rich text plugin replacing the built in rich text editor, and update any references to `'file'` below as the original example plugin replaces the file picker